Automattic/pocket-casts-ios
# Add new protocol and method to sort by name and uuid

Fixes #1959

This PR adds a new method to sort by name and fallback to uuid if names are equal. To conform Podcast and FOlder, I created a new protocol `Sortable`.

## To test

- Make sure you have 2 or more different Podcasts with the same name
- If you don't search for `SUCHT & SÜCHTIG`
- Add them and goto your grid view
- Sort by name
- Check the order
- Whenever the grid reloads (from background, next launch, pull down, etc...) the order should always be the same

## Checklist
